packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -770,6 +788,16 @@ export class Project extends ProjectBase {
     }
   }
 
+  /**
+   * Adds a fileSystemLocation to the Project.
+   *
+   * @param fileSystemLocation the fileSystemLocation to add
+   */
+  public addFileSystemLocations(fileSystemLocation: IFileSystemLocation): void {
```

Comment:
Oh, one more thing - rename this method to `addFileSystemLocation` (no `s` at the end).